lib/tasks/wos.rake

```diff
@@ -5,15 +5,19 @@ namespace :wos do
   end
 
   desc 'Update harvest from Web of Science, for all authors'
-  task harvest_authors_update: :environment do
-    WebOfScience.harvester.harvest_all(update: true)
+  task :harvest_authors_update, [:symbolicTimeSpan] => :environment do
```

Comment:
Need to get the args into the block, like `do |t, args|`.  Right now it will never register the incoming argument and always use the default.  The `args` will be a `Rake::Arguments` object, so you might consider using the conventional `.with_defaults()` call.  Example [here](http://ruby.zigzo.com/2014/08/29/rake-tasks-arguments-and-default-values/).

Reply:
Thanks, done